kg_microbe/download.py

```diff
@@ -1,5 +1,5 @@
 """Download resources from YAML file."""
-from kghub_downloader.download_utils import download_from_yaml  # type: ignore
+from kg_microbe.utils.download_utils import download_from_yaml
```

Comment:
If you're using a local version of the downloader than please open a PR with the changed version on the kghub-downloader repo so we can use the updated version in other KG-Hub repos!

Reply:
Updated download_utils so it wouldn't affect existing code in kghub-downloader, and I will create a PR for that soon

Reply:
This needs to be rolled back to 
```
from kghub_downloader.download_utils import download_from_yaml
```
This is a standard downloader we use for all KGs. @bsantan , I'd recommend removing local code from `kg_microbe.utils.download_util` and make a PR in kghub_downloader so others in the community could benefit from the awesome code!

Reply:
I have started a new PR : https://github.com/Knowledge-Graph-Hub/kg-microbe/pull/106

Please feel free to take over this PR